test/queries/validator_score_query_test.rb

```diff
@@ -3,40 +3,33 @@
 # Query Objects for searching for ValidatorHistory relations and objects
 class ValidatorScoreQueryTest < ActiveSupport::TestCase
   def setup
-    super
-
     batch_uuid = create(:batch).uuid
-    srand(123_456)
-    @validators = [
-      create(:validator, :with_score),
-      create(:validator, :with_score),
-      create(:validator, :with_score),
-      create(:validator, :with_score),
-      create(:validator, :with_score),
-      create(:validator, :with_score)
-    ]
-
-    @validator_histories = @validators.map do |validator|
-      create(:validator_history, batch_uuid: batch_uuid,
-                                 account: validator.account, root_block: 2,
-                                 last_vote: 21, active_stake: 10)
+    
+    6.times do |n|
+      create(:validator, :with_score, account: "validator_#{n}")
     end
-    @validator_scores = @validators.map(&:score)
-    @validator_scores.last.update(active_stake: nil)
 
-    @vsq = ValidatorScoreQuery.new('testnet', batch_uuid)
-  end
+    @validators = Validator.all
 
-  def teardown
-    super
+    @validator_histories = @validators.map do |validator|
```

Comment:
```suggestion
    @validator_histories = @validators.each do |validator|
```

Besides looks like `@validator_histories` is not used anywhere.